swagger.yml

```diff
@@ -3004,6 +3112,154 @@ components:
           x-go-name: Status
       type: object
       x-go-package: github.com/TykTechnologies/tyk
+    APIAllCertificates:
+      description:  APIAllCertificates represents a list of certificates
+      properties:
+        certs:
+          items:
+            type: string
+          type: array
+          x-go-name: Certs
+      type: object
+      x-go-package: github.com/TykTechnologies/tyk
+    APIAllCertificateBasics:
+      description: APIAllCertificates represents a list of certificates
```

Comment:
description is same as `APIAllCertificates`, did you forget to put here `APIAllCertificateBasics`?

Reply:
typo, correcting it now :)